js/menu.js

```diff
@@ -0,0 +1,26 @@
+let menuIcon = document.querySelector('.menu-icon')
+let menuLinks = document.querySelector('.menu-links1')
+let btnClose = document.querySelector('.close')
+let menu_a = document.querySelectorAll('.menu-a')
+menuIcon.addEventListener('click', (e)=>{
+    if(menuLinks.style.display = 'none' || btnClose.style.display == 'none'){
+        menuLinks.style.display ='flex'
+        btnClose.style.display = 'block'
+    }else{
+         menuLinks.style.display = 'none'
+    }
+})
+btnClose.addEventListener('click', ()=>{
+    if(menuLinks.style.display = 'flex'){
+        menuLinks.style.display = 'none'
+        btnClose.style.display = 'none'
+    }
+})
```

Comment:
- [ ] Whenever I click the hamburger button on mobile and close it and then move to desktop the links on the menu do not appear, this is happening because the display: none is remaining, kindly check and do that only for mobile.
https://www.awesomescreenshot.com/video/15358807